contracts/MyStrategy.sol

```diff
@@ -400,4 +426,11 @@ contract MyStrategy is BaseStrategy, ReentrancyGuardUpgradeable {
         IERC20Upgradeable(BADGER).safeTransfer(BADGER_TREE, amount);
         _processExtraToken(address(BADGER), amount);
     }
-}
+
+    /// PAYABLE FUNCTIONS ///
+
+    /// @dev Can only receive ether from Hidden Hand
+    receive() external payable {
```

Comment:
Can we have a storage flag to ensure this is called also only by harvest?

Something like `claiming = true`
`receive -> require(claiming)`

Reply:
wdyt?

Reply:
I guess that makes more sense. I'll do that instead

Reply:
How about now? I also removed the eth sweeping function since that probably won't be required now